account_analytic_profit_loss_analysis/models/account_analytic_line.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = 'account.analytic.line'
+
+    pj_id = fields.Many2one(
+        'project.project',
+        string='Related Project',
+        store=True,
+    )
+    project_partner_id = fields.Many2one(
+        related='pj_id.partner_id',
+        string='Customer',
+        store=True,
+        readonly=True,
+    )
```

Comment:
This field is probably redundant to `partner_id`.  Please remove this if it is the case.

Reply:
I have checked the `partner_id` field, it is a related field of `account_id.partner_id`.
It is slightly different from my logic, but I do think it has the same meaning therefore I removed this field and use `partner_id`